### PR TITLE
{data}[intel/2021b] CDO v1.9.10 and downgrade YAXT from intel/2021b to iimpi/2021b

### PR DESCRIPTION
**Pull request by rungitta**
(created using `eb --new-pr`)
requires: 
 - #14935 
 - #14930

### CODE REVIEW

**Comment by rungitta** [comment]
Test report by @rungitta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login-2.fram.sigma2.no - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) CPU E5-2683 v4 @ 2.10GHz, Python 2.7.5
See https://gist.github.com/cea539a7688b4e2701bccd4b7df13390 for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node301.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/86085ea2fde26aab218d08f27734d1e6 for a full test report.

File: easybuild/easyconfigs/c/CDO/CDO-1.9.10-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'CDO'
+version = '1.9.10'
+
+homepage = 'https://code.zmaw.de/projects/cdo'
+description = """CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data."""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
```
Comment (lexming) [inline easybuild/easyconfigs/c/CDO/CDO-1.9.10-intel-2021b.eb:9]: The toolchain can be lowered to `iimpi` as CDO does not use the numeric libraries Intel MKL

```suggestion
toolchain = {'name': 'iimpi', 'version': '2021b'}
```

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by rungitta** [comment]
Thanks @lexming 
Then I guess the name should also be changed. How do I do that within the PR?

**Comment by rungitta** [comment]
@lexming So to fix the error in the test, do I make a new PR for YAXT with iimpi toolchain or do I somehow tell eb in this easyconfig to look for the version of YAXT built with intel toolchain (don't know how to do this option)?

**Comment by lexming** [comment]
@rungitta right, that's my fault. I didn't realize in #14930 that YAXT does not really need to be in the `intel` toolchain and can be moved to `iimpi`. It uses MPI but not Intel MKL.

Fortunately we can still fix that in this PR, since that easyconfig of YAXT was merged only yesterday and has not hit any EasyBuild release. Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/14936/commits/36c79127a5ff818669aaf6da34f4c626431b40b4

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node315.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/73a34b04853d19e69816752a2a43a936 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @rungitta !

**Pull request merged**